update docs for v0.5.2 (#34)
stackrlabs/docs: docs/pages/build/framework/config.mdx

};
```

where the type definition of `StackrConfig` is as follows:

```ts
interface StackrConfig {
  isSandbox?: boolean;
  deploymentFile?: string;
  sequencer: {
    blockSize: number;
    blockTime: number;
    enableEmptyBlocks?: boolean;
    disableBuilder?: boolean;
  };
  syncer: {
    slotTime?: number;
    vulcanRPC: string;
    L1RPC: string;
    disableVulcanSync?: boolean;
    disableL1Sync?: boolean;
  };
  registryContract: string;
  operator: {
    accounts: Account[];
  };
  domain: {
    name: string;
    version: string;
    salt: string;
  },
  datastore: {
    type?: 'sqlite' | 'postgres' | 'mysql' | 'mariadb';
    uri: string;
  };
  preferredDA?: 'avail' | 'celestia';
  logLevel?: 'log' | 'error' | 'warn' | 'debug' | 'verbose';
}
```

## Parameters

1. `isSandbox`: A boolean flag to indicate if the rollup is running in a sandbox environment. Defauts to `false`.

> When running in a sandbox mode, the rollup creates a temporary sqlite database to store the state and the actions and the blocks are not sent to Vulcan or L1 for verification.

Reviewer: ```suggestion
When running an MRU in sandbox mode, the rollup creates a temporary SQLite database to store data and the blocks are not sent to Vulcan or L1 for verification. Vulcan and L1 Syncers are disabled in sandbox mode and therefore the confirmation you get on an action is `C1`.
```

This file is created automatically during registration of the micro-rollup with the registry contract using `stackr register` command. See [Register](/build/cli/register) for more details.

---

3. `sequencer`: Contains the parameters for the sequencer module.

- `blockSize`: The number of actions that the sequencer picks up in a single block.
- `blockTime`: The time interval after which the sequencer picks up the actions in a block.
- `enableEmptyBlocks`: A flag to allow creation of empty blocks (the one with no actions).
- `disableBuilder`: A flag to indicate if the rollup should not use the builder to generate new blocks. This is useful for functionally generating blocks when needed. Defauts to `false`.

---

4. `syncer`: Contains the parameters for the syncer module.

- `slotTime`: The time interval after which the rollup sends a block to Vulcan for verification and finalization.
- `vulcanRPC`: The RPC endpoint of the Vulcan chain. Can be found in [Network Configuration](/build/references/providers-and-rpc).
- `L1RPC`: The RPC endpoint of the L1 chain. Can be found in [Network Configuration](/build/references/providers-and-rpc).

> - `disableVulcanSync`: A flag to indicate if the rollup should not sync with Vulcan. This happens by default in sandbox mode. Defauts to `false`.

Reviewer: ```suggestion
- `disableVulcanSync`: A flag to indicate if the rollup should not sync with Vulcan. Defaults to `false`, except in Sandbox mode, where it defaults to `true`.
```

> - `disableL1Sync`: A flag to indicate if the rollup should not sync with the L1 chain. This happens by default in sandbox mode. Defauts to `false`.

Reviewer: ```suggestion
- `disableL1Sync`: A flag to indicate if the rollup should not sync with the L1 chain. Defaults to `false`, except in Sandbox mode, where it defaults to `true`.
```

---

5. `operator`: Contains the parameters for the operator module.

- `accounts`: Contains the private key of the operator account. This private key is used for registration and batch signing

---

6. `domain`: Contains the parameters for the EIP-712 domain.

- `name`: The name of the micro-rollup
- `version`: The version of the micro-rollup
- `salt`: A random salt value

> The other fields, `chainId` and `verifyingContract`, required by EIP-712 domain definition are taken automatically from the given deployment file.

Reviewer: Link https://eips.ethereum.org/EIPS/eip-712 here

---

7. `datastore`: Contains the parameters for the datastore module.

This is used for local storage of the micro-rollup state, actions and blocks

---

8. `registryContract`: The address of the registry contract.

> 

Reviewer: ```suggestion
```
